src/broker/__tests__/addPartitionsToTxn.spec.js

```diff
@@ -86,7 +86,7 @@ describe('Broker > AddPartitionsToTxn', () => {
     await expect(
       broker.addPartitionsToTxn({
         transactionalId,
-        producerId: 'foo',
+        producerId: '123456789',
```

Comment:
I looked at whether we should check for this early and validate that the producer id is something that can be converted to a bigint, but since it's not a public api, I think it's fine to leave as it is. The change here is just to make the operation fail for the right reason.